### PR TITLE
Use unit types from Unitful.jl

### PR DESCRIPTION
**Pull request by 8me**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -4,9 +4,9 @@ authors = ["Johannes Schumann", "Tamas Gal"]
 version = "0.1.0"
 
 [deps]
-CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
```
Comment (tamasgal) [inline Project.toml]: I already removed them in master, so don't be surprised. They broke the Travis build due to whatever weird error ;)

File: src/Particles.jl [hunk 2]
```diff
@@ -26,17 +29,20 @@ end
     ChargeInv = 2
 end
 
-struct MeasuredValue{T}
-    value::T
-    lower_limit::T
-    upper_limit::T
+struct MeasuredValue{D} 
+    value::Quantity{T1,D,U1} where {T1 <: Real, U1 <: Unitful.Units}
```
Comment (tamasgal) [inline src/Particles.jl]: Ah I see, the limits can have different unit-prefixes? Is it guaranteed that we do not mix `cm` with `kg`? ;-)

File: Project.toml (hunk 1 above)
Comment (8me) [reply Project.toml]: I've seen your commit, but as it affects just two lines I thought a rebase is kinda over the top ;-)

Comment (tamasgal) [reply Project.toml]: It seems that it has no merge conflicts, so all good ;)

**Comment by tamasgal** [comment]
Btw: `Merging #2 into master will decrease coverage by 3.92%`. 🙈

**Pull request merged**